### PR TITLE
UserGuide.adoc: Update Quick start and Find

### LINKED ISSUE #97: Update Quick Start section for User Guide

### LINKED ISSUE #109: Update find feature in User Guide
Update find feature in User Guide

### PR DESCRIPTION
**Pull request by khsc96**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -169,6 +186,7 @@ Examples:
 * `import orders.csv` +
 Import the contents of the csv file `orders.csv` to Delino.
 
+[[List]]
```
Comment (Exeexe93) [inline docs/UserGuide.adoc]: Please change to [[list]].

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -103,48 +103,65 @@ Deletes the 1st person in the results of the `find` command.
...
-* You can edit the details of the delivery order by providing the order transaction id, the field you want to change followed by the updated value.
+
+* You can edit the details of the delivery order by providing the index of it shown on the status bar, followed by the field you want to change and lastly the updated value.
 
 ==== Format
-[none]
-* `edit *TRANSACTION_ID* *FIELD_TO_CHANGE* *VALUE*`
+
+[TIP]
+Use the <<List, `list`>> command to see which delivery order you want to edit.
```
Comment (Exeexe93) [inline docs/UserGuide.adoc]: Please change to <<list, `list`>>

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -23,8 +23,8 @@ AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for mana
 == Quick Start
 
 .  Ensure you have Java `11` or above installed in your Computer.
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
+.  Download the latest `Delino.jar` link:{repoURL}/releases[here].
+.  Copy the file to the folder you want to use as the home folder for youDelino.
```
Comment (Exeexe93) [inline docs/UserGuide.adoc]: Is there a typo error for 

> youDelino?

Is it you Delino?

**Review by Exeexe93 — CHANGES_REQUESTED** [review]
Please resolve the merge conflict and push your updated version ! Do amend those I comment. Thanks!

File: docs/UserGuide.adoc (hunk 3 above)
Comment (Cherweijie) [reply docs/UserGuide.adoc]: I think should just be Delino right?

**Review by Cherweijie — COMMENTED** [review]
I think besides the minor formatting issue and typo, it looks good to merge!

**Review by Cherweijie — APPROVED** [review]
LGTM

**Review by Exeexe93 — APPROVED** [review]
LGTM

**Pull request merged**